README.md

```diff
@@ -269,6 +269,14 @@ profile.registerType(classToRegister, locationToRegister, 'container', 'listed')
     // querying the registry now will include the new container
     profile.typeRegistryForClass(vocab.sioc('Post'))
   })
+
+// To remove the same class from registry:
+var classToRemove = vocab.sioc('Post')
+profile.unregisterType(classToRemove, 'listed')
```

Comment:
I was thinking about that too. My thought was to make it a string to have it be more explicit (vs having to remember which parameter the `true` represents). 
I'm not committed to it, though, happy to refactor the code to use `true`, both for register & unregister.